### DISCUSSION TITLE
RDS for XA60

### DISCUSSION THREAD

**Discussion by skwarfield** [yarra-framework/yarra-framework.github.io#6] [Q&A] [answered]
We are in the process of scheduling an upgrade to XA60.
Will Yarra be available for XA60 ?

**Comment by jfpva** [comment]
Same question, but for XA61. 
One of our scanners has just made the leap from VE platform all the way to XA61.

**Comment by jberrebi** [comment]
Hi,
We are also going to upgrade to XA60A in August and you love to be able to use RDS.

**Comment by tblock79** [comment, marked as answer]
A new version of the Yarra Client with support for XA60/XA61 has been released and is now available in the Download section of the website.
